app/mm-redux/actions/channels.ts

```diff
@@ -1517,6 +1517,14 @@ export function getChannelMemberCountsByGroup(channelId: string, includeTimezone
     };
 }
 
+export function isChannelMember(channelId: string): ActionFunc {
```

Comment:
Shouldn't this be in the selectors file? I don't think this function fits well into the actions.

Reply:
Thanks! I have moved it to utils.